pkg/option/config.go

```diff
@@ -902,6 +902,14 @@ const (
 
 	// EnableBPFBypassFIBLookup instructs Cilium to enable the FIB lookup bypass optimization for nodeport reverse NAT handling.
 	EnableBPFBypassFIBLookup = "bpf-lb-bypass-fib-lookup"
+
+	// DisableCiliumNetworkPolicyCRDName is the name of the option to disable
+	// use of the CNP and CCNP CRD
+	DisableCiliumNetworkPolicyCRDName = "disable-network-policy-crd"
+
+	// EnableEgressRevnatAtHostOnly enables reverse NAT translation at node's
+	// host interface instead of veth interface on egress datapath.
+	EnableEgressRevnatAtHostOnly = "enable-egress-revnat-at-host-only"
```

Comment:
ditto

Reply:
Not sure about the comment, Can you explain more?

Reply:
Ah I read it wrong, this is the new flag we need, so nvm :)